test/mocks/in-memory-mongo-store.ts

```diff
@@ -1,6 +1,6 @@
 import { MongoMemoryServer } from 'mongodb-memory-server';
 import { connect, Mongoose as M } from 'mongoose';
-import { Mongoose } from '@storage';
+import { Mongoose } from '../../src/storage';
```

Comment:
Can you document / comment on reasons why aliases (like - `@storage` / `@typings` etc) are not working? There might be some issue with our boilerplate code and we should investigate on how to fix it.